pcsx2/Docs/GameIndex.md

```diff
@@ -49,8 +49,10 @@ SERIAL-12345: # !required! Serial number for the game, this is how games are loo
   # The value of the speedhacks is assumed to be an integer,
   # but at the time of writing speedhacks are effectively booleans (0/1)
```

Comment:
If you go add stuff like EEcyclerate and even cycleskip here, the text isn't exactly correct here anymore, needs addendum for negative and positive numbers.